### PR TITLE
GEOMESA-2090 Displaying Arrow hints in query audits

### PR DESCRIPTION
**Pull request by elahrvivaz**
* Allowing filter by arrow query in web endpoint
* Changing storage of query hints to match udpated names

Signed-off-by: Emilio Lahr-Vivaz <elahrvivaz@ccri.com>

### CODE REVIEW

File: geomesa-accumulo/geomesa-accumulo-spark-runtime/pom.xml [hunk 1]
```diff
@@ -81,9 +81,7 @@
         <dependency>
             <groupId>org.locationtech.geomesa</groupId>
             <artifactId>geomesa-accumulo-datastore_${scala.binary.version}</artifactId>
-            <version>${project.version}</version>
             <classifier>tests</classifier>
-            <scope>test</scope>
```
Comment (jnh5y) [inline geomesa-accumulo/geomesa-accumulo-spark-runtime/pom.xml]: Did this dependency move to compile?

File: geomesa-web/geomesa-web-data/pom.xml [hunk 2]
```diff
@@ -145,6 +145,23 @@
...
+
+        <!-- test dependencies -->
+
+        <dependency>
+            <groupId>org.specs2</groupId>
+            <artifactId>specs2_${scala.binary.version}</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.scalatra</groupId>
+            <artifactId>scalatra-specs2_2.11</artifactId>
```
Comment (jnh5y) [inline geomesa-web/geomesa-web-data/pom.xml]: Should the 2.11 be ${scala.binary.version}?

File: geomesa-accumulo/geomesa-accumulo-spark-runtime/pom.xml (hunk 1 above)
Comment (elahrvivaz) [reply geomesa-accumulo/geomesa-accumulo-spark-runtime/pom.xml]: no, we were just declaring it in a weird way - i standardized it into the main pom.xml

File: geomesa-web/geomesa-web-data/pom.xml (hunk 2 above)
Comment (elahrvivaz) [reply geomesa-web/geomesa-web-data/pom.xml]: yeah, good catch

File: geomesa-accumulo/geomesa-accumulo-spark-runtime/pom.xml (hunk 1 above)
Comment (jnh5y) [reply geomesa-accumulo/geomesa-accumulo-spark-runtime/pom.xml]: Ah! ok!

File: geomesa-web/geomesa-web-data/pom.xml (hunk 2 above)
Comment (elahrvivaz) [reply geomesa-web/geomesa-web-data/pom.xml]: fixed

**Pull request merged**